librz/core/cmd/cmd_debug.c

```diff
@@ -2404,6 +2404,19 @@ RZ_IPI RzCmdStatus rz_cmd_debug_continue_call_handler(RzCore *core, int argc, co
 	return RZ_CMD_STATUS_OK;
 }
 
+// dcco
+RZ_IPI RzCmdStatus rz_cmd_debug_continue_call_over_handler(RzCore *core, int argc, const char **argv) {
+	CMD_CHECK_DEBUG_DEAD(core);
+	rz_cons_break_push(rz_core_static_debug_stop, core->dbg);
+	rz_reg_arena_swap(core->dbg->reg, true);
+
+	rz_debug_continue_until_optype(core->dbg, RZ_ANALYSIS_OP_TYPE_CALL, 1);
```

Comment:
i wonder if it makes sense to have this or just add an argument to `dcc` and have `dcc over` which sets the last arg of this to `1` instead of `0`.

@ret2libc @XVilka

Reply:
I think separate command is fine